.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @khwilson
\ No newline at end of file
+* @bashtage @khwilson
```

Comment:
I think this needs to be in list form:

```
* @bashtage
* @khwilson
```

Reply:
I think the current is correct:


See, e.g., 

https://github.com/conda-forge/statsmodels-feedstock/blob/master/recipe/meta.yaml